eatngo-infra/src/main/resources/application.yml

```
@@ -0,0 +1,8 @@
+cloud:
+  aws:
+    region:
+      static: us-east-1
+
+aws:
+  s3:
+    bucket: eatgoeatgo
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)